pandevice/__init__.py

```diff
@@ -136,24 +143,45 @@ def subrelease_num(self):
     def __repr__ (self):
         return "PanOSVersion ('%s')" % str(self)
 
-    def __cmp__ (self, other):
-        if isinstance(other, basestring):
-            other = PanOSVersion(other)
-
-        # Compare subreleases if number part of version is the same
-        if cmp(self.mainrelease, other.mainrelease) == 0:
-            if self.subrelease_type == 'c' and other.subrelease_type != 'c':
-                return -1
-            elif self.subrelease_type == 'b' and other.subrelease is None:
-                return -1
-            elif self.subrelease_type != 'c' and other.subrelease_type == 'c':
-                return 1
-            elif self.subrelease is None and other.subrelease_type == 'b':
-                return 1
-
-        return cmp(self.version, other.version)
-
-
+    def __lt__(self, other):
```

Comment:
I believe I have fixed the issue. I added a more extensive test file to check and it seems to be working. Will add more tests tomorrow.